source/main.py

```diff
@@ -81,7 +81,7 @@ def start_invite_key(text: str):
 
 
 def generate_invite_link(invite_key):
-    return 'https://t.me/hwhelperbot?start=' + invite_key
```

Comment:
maybe I can delete 'https://' prefix and make link shorter

Reply:
maybe